std/math.d

```diff
@@ -4066,11 +4066,11 @@ private:
         // ARM FPSCR is a 32bit register
         enum : int
         {
-            INEXACT_MASK   = 0x1000,
-            UNDERFLOW_MASK = 0x0800,
-            OVERFLOW_MASK  = 0x0400,
-            DIVBYZERO_MASK = 0x0200,
-            INVALID_MASK   = 0x0100
+            INEXACT_MASK   = 0x10,
+            UNDERFLOW_MASK = 0x08,
+            OVERFLOW_MASK  = 0x04,
+            DIVBYZERO_MASK = 0x02,
+            INVALID_MASK   = 0x01
```

Comment:
Seems like the change is correct though, as the docs say `IEEE exception status flags ('sticky bits')` and `These flags indicate that an exceptional floating-point condition has occurred`.
Someone had the bad idea to append this mask suffix.